examples/minitest_helper.rb

```diff
@@ -35,4 +35,5 @@ def use_natural_assertions_if_supported(enabled=true)
 end
 
 Minitest::Spec.send(:include, GivenAssertions)
+Minitest::Test.send(:include, GivenAssertions)
```

Comment:
So as far as I could tell GivenAssertions are only used in his examples which is why it's mixed in via the helper.  It seems quite odd, no?  I was using part of his given_spec to test that Given/When/Then would work within nested blocks and contexts using the same code he was using.  I had to add those assertions to Minitest::Test because if I make my test inherit from Minitest::Spec then I can't replicate the bug where the DSL isn't mixed into ActiveSupport:TestCase.  The one thing I forgot is that Minitest::Spec inherits from Minitest::Test so only one of these lines was required.  I'm fixing that.